examples/src/iframe.tsx

```diff
@@ -0,0 +1,43 @@
+// @ts-nocheck
```

Comment:
I ran out of time to sort some typing issues in this file, will come back to it and fix if general approach is approved.